### ISSUE TITLE
New file is created when Npp is closing. Bug or not?

### ISSUE THREAD

**Issue by Rikk** [notepad-plus-plus/notepad-plus-plus#234]
I recorded [**this screen capture**](http://ri.cardo.nom.br/etc/NppExitBug.wmv) to exemplify the following:
I'm not sure if this is wrong behavior, but when Npp is closing, right after the last file is closed and before Npp window vanishes, a new file ("new 1" on window title) is created with default options (look the status bar changing to the default encoding values).

**Comment by milipili** [comment]
As reported by @Eldaw, this bug should no longer be directly visible since ba5d36e. The routine `LockWindowUpdate` (or any new and better implementation) should be disabled to reproduce this issue.

**Comment by Rikk** [comment]
Ok. Was trying to find what glitch that commit tried to fix...

**Comment by Rikk** [comment]
No, https://github.com/notepad-plus-plus/notepad-plus-plus/commit/ba5d36e2bfebc7325b6777ab2d0b54d7e49e2620 does not fix this issue, I can still see it happen.
